packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -71,5 +73,14 @@ export const comboboxTemplate: FoundationElementTemplate<
                 })}
             ></slot>
         </div>
+        <label
+            part="label"
+            class="${x =>
+                x.slottedLabel && x.slottedLabel.length
+                    ? "label"
+                    : "label label__hidden"}"
```

Comment:
This pattern has always bothered me since it duplicates the `label` class in both cases (likely to avoid outputting an extra space when not hidden). How about this instead?

```suggestion
        class="${x =>
            ["label", !x.slottedLabel?.length && "label__hidden"]
                .filter(Boolean)
                .join(" ")}"
```

Reply:
Or:

```
class="label${ x => ["", " label_hidden"][+!x.slottedLabel?.length]}"
```

Reply:
I think it's clever, but I also think it's more costly. In the absence of jsperf: https://jsben.ch/QA729

If we can come up with a better pattern, I'm good with that but it needs to be more performant than what is currently here.

Reply:
Thanks Rob - Can you move this into an issue perhaps with a proposal for where it might live as well as the possible perf cost related to current implementation? I'm open to making a change to how we handle this, but I'd like to not do so as part of this PR because it's changing a pattern across the repo. Let's take this to an issue and we can hear from others + the community on this.

Reply:
Absolutely. I only brought it up to start a discussion